HISTORY.rst

```diff
@@ -5,6 +5,13 @@ Release History
 
 unreleased
 +++++++++++++++
+**IoT Hub updates**
```

Comment:
Please merge this section with the lower unreleased IoT Hub section with the device-twin list entry.